server/services/decisionTreeService.ts

```diff
@@ -111,17 +111,26 @@ export default class DecisionTreeService {
   async answerDataDetails(answerData: AnswerData, user: User): Promise<AnswerDataDetails> {
     const [victimOtherPerson, victimPrisoner, victimStaff] = await Promise.all([
       answerData.victimOtherPersonsName,
-      answerData.victimPrisonersNumber &&
-        this.placeOnReportService.getPrisonerDetails(answerData.victimPrisonersNumber, user),
+      this.getPrisonerDetails(answerData, user),
       answerData.victimStaffUsername && this.userService.getStaffFromUsername(answerData.victimStaffUsername, user),
     ])
     return {
       victimOtherPerson,
       victimPrisoner,
       victimStaff,
+      victimPrisonerNumber: answerData.victimPrisonersNumber,
     }
   }
 
+  private async getPrisonerDetails(answerData: AnswerData, user: User): Promise<PrisonerResultSummary> {
+    // The victimOtherPersonsName will be set if the prisoner is from outside the establishment,
+    // in which case we do not want to get the extra prisoner information
+    if (!answerData.victimPrisonersNumber || answerData.victimOtherPersonsName) {
```

Comment:
This is not nice, but the alternatives involve quite a lot of work.

I have taken a pragmatic approach - make it work with what we have and add a comment and some tests.